src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -24,16 +24,16 @@ class FormIntroButtons extends React.Component {
 
   handleLoadPrefill = () => {
     if (this.props.prefillAvailable) {
-      this.handleLoadForm(true);
+      this.props.fetchInProgressForm(this.props.formId, this.props.migrations, true);
```

Comment:
handleLoadForm is always passed an event as an argument by `ProgressButton`, so this works better for passing the prefill parameter.